src/Sylius/Behat/Context/Setup/PromotionContext.php

```diff
@@ -130,11 +130,44 @@ public function thereIsPromotionWithCoupon($promotionName, $couponCode, $usageLi
         $promotion->setCouponBased(true);
 
         $this->promotionRepository->add($promotion);
+
         $this->sharedStorage->set('promotion', $promotion);
+        $this->sharedStorage->set('promotion_coupon', $coupon);
         $this->sharedStorage->set('coupon', $coupon);
     }
 
     /**
+     * @Given /^(this promotion) has already expired$/
+     */
+    public function thisPromotionHasExpired(PromotionInterface $promotion)
+    {
+        $promotion->setEndsAt(new \DateTime('1 day ago'));
+
+        $this->objectManager->flush();
+    }
+
+    /**
+     * @Given /^(this promotion coupon) has already expired$/
+     */
+    public function thisPromotionCouponHasExpired(CouponInterface $coupon)
+    {
+        $coupon->setExpiresAt(new \DateTime('1 day ago'));
+
+        $this->objectManager->flush();
+    }
+
+    /**
+     * @Given /^(this promotion coupon) has already reached its usage limit$/
+     */
+    public function thisPromotionCouponHasReachedItsUsageLimit(CouponInterface $coupon)
```

Comment:
I guess that you can merge this method with: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Behat/Context/Setup/PromotionContext.php#L460